.github/workflows/ci.yml

```diff
@@ -15,20 +15,15 @@ jobs:
       uses: actions/setup-python@v4
       with:
         python-version: "3.11"
-        cache: 'pip'
-        cache-dependency-path: '**/pyproject.toml'
-    - name: Install dependencies
-      run: |
-        python -m pip install --upgrade pip
-        python -m pip install tox
+    - name: Install uv
+      run: pip install uv
     - name: Run Lint
-      run: tox -e lint
+      run: |
+        make lint
 
   test:
-    # Dependency on linting to avoid running our expensive matrix test for nothing
```

Comment:
Why do you remove this comment and the other ones below?

Reply:
I'm actually not sure! (tbh I don't remember). Let's add them back.